### ISSUE TITLE
Method used for the results table

### ISSUE THREAD

**Issue by Pangoraw** [vturrisi/solo-learn#221]
I was wondering what evaluation method is used for the [performance tables](https://github.com/vturrisi/solo-learn#results) in the readme and could not figure it out from the json configuration file.

Thanks for all the work :+1:

**Comment by vturrisi** [comment]
Hey,
All results reported used linear evaluation (where you pretrain the backbone in a self-supervised way, freeze it and then train a linear classifier on top of it using labelled data). For cifar, we report online linear evaluation (the backbone and the linear layer are trained at the same time, but the gradients of the linear layer are not used to update the backbone at all), as we saw that this resulted in slightly better performance. For imagenet/imagenet100, we report both values.

Recently, we added a way to evaluate with knn, so you might also wanna experiment with this. Note that performance will be lower.

**Comment by Pangoraw** [comment]
Awesome, thanks for the quick and very detailed answer!
